website/src/hooks/hubSpot.ts

```diff
@@ -85,16 +85,10 @@ interface HookProps {
 
 const loadHubSpotScript = (): HTMLScriptElement | Element => {
     const hubSpotScript = '//js.hsforms.net/forms/v2.js'
-    const script = document.querySelector(`script[src="${hubSpotScript}"]`)
-
-    if (!script) {
-        const scriptElement = document.createElement('script')
-        scriptElement.src = hubSpotScript
-        document.head.append(scriptElement)
-        return scriptElement
-    }
-
-    return script
+    const scriptElement = document.createElement('script')
```

Comment:
@bretthayes I don't **think** we need to do this for ChiliPiper, but lmk if you disagree.

Reply:
Yeah, this goes along with the same process for HubSpot since we don't want to append the script multiple times if it's already present on the page.

Reply:
100%